File: ws/Vaca/u2/WS24Zoo/Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Feline.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Luis Vaca, DCCO-ESPE
 */
public abstract class Feline extends Mammal{

    public Feline(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

}

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Feline.js
const Mammal = require('../ec.edu.espe.Zoo.model/Mammal');

class Feline extends Mammal {
  constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
    super(id, description, dateOnborn, gender, numberOfBones, cage);
  }
}

module.exports = Feline;
